templates/layouts/_base.html

```diff
@@ -73,32 +73,36 @@
 
 
 {% block head %}
-  {% if google_tag_manager_id and google_tag_manager_auth %}
-    <!--
-    Information such as events or variables can be passed to Google Tag Manager via the data layer,
-    and triggers can be set up in Google Tag Manager
-    -->
+  {% if google_tag_id %}
     <script nonce="{{ csp_nonce }}">
       dataLayer = {{ data_layer | tojson }}
     </script>
 
-    <!-- Google Tag Manager -->
+    <!-- Google tag (gtag.js) -->
     <script nonce="{{ csp_nonce }}">
       function loadGTM() {
```

Comment:
We should rename to `loadGoogleTagScript`

Reply:
Updated 👍